.github/workflows/github-pr.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
         with:
           repository: responsivedev/sindri
           path: sindri
-          ref: refs/heads/main
+          ref: refs/heads/SYSTEM-TEST-improve-logging
```

Comment:
@ableegoldman is this intentional?

Reply:
ah I just saw this is a temporary commit in the history - I thought this was already merged but it isn't

Reply:
ah, shoot. didn't mean to commit that -- thanks

Reply:
>  I thought this was already merged but it isn't

Yeah sorry I was overconfident at first and said I was going to merge it ASAP since it was pretty late, but I always wait for the tests to pass -- and they did not. This was a remnant of my debugging because I didn't know about the system test logs in DD lol